src/screens/toDoList/FlatListToDoList.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {FlatList, ListRenderItem, Modal, Text, View} from 'react-native';
+import ToDoListRow from './components/ToDoListRow';
```

Comment:
```suggestion
import {ToDoListRow} from './components';
```
export ToDoListRow from index.ts